packages/client/tests/functional/batch-transaction-isolation-level/tests.ts

```diff
@@ -83,7 +83,7 @@ testMatrix.setupTestSuite(
       await prisma.$transaction([prisma.user.findFirst({}), prisma.user.findFirst({})])
 
       const match = queries.find((q) => q.includes('SET TRANSACTION ISOLATION LEVEL'))
-      if (isSqlServer) {
+      if (isSqlServer && driverAdapter === undefined) {
```

Comment:
Note from call: if the driver adapter uses phantom queries, they don't get logged through the regular logging.
`queries` is an array of logged queries.